src/applications/disability-benefits/all-claims/pages/conclusionUnemployability.js

```diff
@@ -0,0 +1,10 @@
+export const uiSchema = {
+  'ui:title': ' ',
+  'ui:description':
+    'Thank you for taking the time to answer our questions. The information you provided will help us process your claim.',
+};
+
+export const schema = {
```

Comment:
Because this schema doesn't have any properties, and the corresponding uiSchema is super simple, you can totally just put this in the formConfig itself. We've done this for some other pages like `newDisabilities`

Reply:
@markgreenburg : Thank you! I put this in the formConfig. Can you review again?